labs/direct-state-transfer_perun.md

```diff
@@ -0,0 +1,62 @@
+# Lab Name
+
+Direct State Transfer / Perun
+
+# Short Description
+
+Blockchain-agnostic State Channels framework, written in Go.
+
+# Scope of Lab
+
+Perun / Direct State Transfer is a Go implementation of the [Perun state
```

Comment:
What blockchain platforms is this lab looking to support?

Reply:
We try to build the software blockchain agnostic, with the ledger specific parts being implemented in different backends. Currently the implementation supports Ethereum and our focus of development is to further implement the Perun state channel protocol and node. Once that's in place, we'd certainly want to ensure it works with Besu and support other blockchains depending on community interest  / feedback.

Of course, if there is immediate interest of someone wanting to contribute support for another blockchain, we'd also be happy to help with implementing the corresponding blockchain backend.